addons/nametags/XEH_postInit.sqf

```diff
@@ -1,7 +1,9 @@
 // by commy2 and esteldunedain
 #include "script_component.hpp"
 
-[] call FUNC(initIsSpeaking);
+if (GVAR(showPlayerNames) > 0) then {
+    [] call FUNC(initIsSpeaking);
```

Comment:
Same here; it's probably better to always init this regardless of it's going to be used or not